test/check-application

```diff
@@ -14,35 +14,46 @@ import testlib
 
 
 class TestApplication(testlib.MachineCase):
+
     def testBasic(self):
         b = self.browser
         m = self.machine
 
         self.login_and_go("/podman")
+        b.wait_present("#app")
+        b.wait_present(".content-filter div")
+        b.wait_present("#containers-images")
+        b.wait_in_text("#containers-images", "busybox:latest")
+
+        # show image listing toggle
+        b.wait_present('#containers-images tr:contains("busybox:latest")')
+        b.wait_visible('#containers-images tr:contains("busybox:latest")')
+        b.click('#containers-images tbody tr:contains("busybox:latest") td.listing-ct-toggle')
+        b.wait_visible('#containers-images tbody tr:contains("busybox:latest") + tr button.btn-delete')
+
+        # show all containers to interact with stopped ones
+        m.execute("podman run -d --name swamped-crate busybox sh")
```

Comment:
This cannot work, the container will exit immediately. This is why in my initial tests I ran `sleep 10000` in it.

Reply:
Oh, I see, it's supposed to be in the "exited" state, as you switch to "everything".